src/app/components/Sorter/Sorter.jsx

```diff
@@ -61,31 +75,54 @@ class Sorter extends React.Component {
     this.setState({ active: false });
   }
 
-  handleOnClickOut() {
-    if (this.state.active) {
-      this.setState({ active: false });
-    }
+  /**
+   * renderResultsSort()
+   * The fuction that makes renders the sort options.
+   *
+   * @return {HTML Element}
+   */
+  renderResultsSort() {
+    return sortingOpts.map((d, i) => (
+      <option value={d.val} key={i}>
+        {d.label}
+      </option>
+    ));
   }
 
   render() {
+    const keywords = this.props.keywords || '';
```

Comment:
This should be `searchKeywords` and if you're going to set default `''` then do it as the `defaultProps` object.